### PR TITLE
link some actions from readme and add libera addresses

### PR DESCRIPTION
**Pull request by obernardovieira**
This PR adds Libera issuer's addresses as well as some links to other parts of the documentation that I felt would make it simpler.

### CODE REVIEW

**Comment by 0xarthurxyz** [comment]
Thanks for the suggestions @obernardovieira ! Really appreciate it 🙏

File: asv2/README.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ The following steps use the Celo [ContractKit](https://docs.celo.org/developer/c
 
 ## 🚀 Examples
 
-The best place to get started is the [examples](examples) folder, which contains sample scripts that implement registration and lookups, using [ContractKit](https://docs.celo.org/developer/contractkit), [Ethers](https://ethers.org/), and [web3.js](https://web3js.readthedocs.io/en/v1.8.1/).
+The best place to get started is the [examples](examples) folder, which contains sample scripts that implement registration and lookups, using [ContractKit](https://docs.celo.org/developer/contractkit), [Ethers](https://ethers.org/), and [web3.js](https://web3js.readthedocs.io/en/v1.8.1/). See [Runtime Environments](privacy.md#runtime-environments) to find out about BLS to each environment.
```
Comment (isabellewei) [inline asv2/README.md:132]:
```suggestion
The best place to get started is the [examples](examples) folder, which contains sample scripts that implement registration and lookups, using [ContractKit](https://docs.celo.org/developer/contractkit), [Ethers](https://ethers.org/), and [web3.js](https://web3js.readthedocs.io/en/v1.8.1/). Make sure to also consult the [runtime environments](privacy.md#runtime-environments) section to ensure you're using the correct implementation for your project.
```

File: asv2/README.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ The following steps use the Celo [ContractKit](https://docs.celo.org/developer/c
     npm install @celo/identity
     ```
 
-2. Set up your issuer, which is the account registering attestations. When a user requests for the issuer to register an attestation, the issuer should [verify](protocol.md#verification) somehow that the user owns their identifier (ex. SMS verification for phone number identifiers).
+2. Set up your issuer (read "Authentication" section in [privacy.md](privacy.md#authentication)), which is the account registering attestations. When a user requests for the issuer to register an attestation, the issuer should [verify](protocol.md#verification) somehow that the user owns their identifier (ex. SMS verification for phone number identifiers).
```
Comment (isabellewei) [inline asv2/README.md:38]: I think this should be linked in the next step (step 3) instead, as interactions with ODIS and using the auth signer isn't actually done here

Comment (isabellewei) [reply asv2/README.md:38]: I can't seem to add the suggestion directly in this PR, but I would add to step 3:

> See the privacy doc for details on [authentication](privacy.md#authentication) and the [service context](privacy.md#service-context)

**Review by isabellewei — APPROVED** [review]
Thank you for taking the time to make these edits! Greatly appreciated ☺️

**Pull request merged**